projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -345,9 +345,9 @@ export class IgxGridComponent extends IgxGridBaseDirective implements GridType,
      * this.grid1.totalItemCount = 55;
      * ```
      */
+    @Input()
     public set totalItemCount(count) {
         this.verticalScrollContainer.totalItemCount = count;
-        this.cdr.detectChanges();
```

Comment:
With this being an Input, the explicit detectChanges call should no longer be needed.
The dev demos for remote virt don't work for me locally, so will still need help with verification of that.
At least the tests pass, but those have their forced checks too often.